src/DependencyInjection/SonataPageExtension.php

```diff
@@ -168,15 +176,22 @@ public function registerParameters(ContainerBuilder $container, array $config)
     }
 
     /**
+     * NEXT_MAJOR: Remove this method.
+     *
      * Registers doctrine mapping on concrete page entities.
      */
     public function registerDoctrineMapping(array $config)
     {
+        @trigger_error(
+            'Using SonataEasyExtendsBundle is deprecated since sonata-project/timeline-bundle 3.x. Please register SonataDoctrineBundle as a bundle instead.',
```

Comment:
```suggestion
            'Using SonataEasyExtendsBundle is deprecated since sonata-project/page-bundle 3.x. Please register SonataDoctrineBundle as a bundle instead.',
```

Reply:
Fixed 😓